README.md

```diff
@@ -11,7 +11,7 @@ https://hsadmin.trink.com/
 
 * GCC (4.8+)
 * CMake (3.5+) - http://cmake.org/cmake/resources/software.html
-* Boost (1.53+) - http://www.boost.org/users/download/
+* Boost (1.65.1) - http://www.boost.org/users/download/
 * Wt (3.3.4) - http://www.webtoolkit.eu/wt/download
```

Comment:
@trink also realized here we have 3.3.4; the image build recently added uses 3.3.8 and builds -- can we update this as well or was 3.3.4 required for some reason?

Reply:
I have been building with 3.3.9